plex/umbrel-app.yml

```diff
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: plex
+category: Files
+name: Plex
+version: "1.28.1.6104-788f82488"
+tagline: Watch anytime, anywhere with the Plex app.
+description: >-
+  Plex is an American streaming media service and a client–server media player platform, made by Plex, Inc. The Plex Media Server organizes video, audio, and photos from a user's collections and from online services, and streams it to the players. The official clients and unofficial third-party clients run on mobile devices, smart TVs, streaming boxes, and in web apps.
+developer: Plex
+website: https://plex.tv
+dependencies:
+ - simple-torrent
```

Comment:
This dependency is not strictly required for local file playback

Reply:
true, this is just how I use Plex but it's not necessary